Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -16,10 +16,8 @@ def typecheck_args
       EOS
       switch "-q", "--quiet",
              description: "Silence all non-critical errors."
-      switch "--update-definitions",
-             description: "Update Tapioca gem definitions of recently bumped gems"
-      switch "--prune-files-list",
-             description: "Remove deleted filepaths from #{SORBET_FILES_YAML}"
+      switch "--ci",
```

Comment:
Or `--fix`?

Reply:
Or `--update`? 🤷🏻

Reply:
I was thinking we could make `typecheck` part of `brew style`, since it's basically a lint, that's why I suggested `--fix`.

Reply:
@vidusheeamoli or @mistydemeo might have different opinions, but I'm not sure we're at that level of efficiency yet with Sorbet. That said, I'm not sure we ever will be unless we three get some help solving the issues - and that can only be done if it reaches more devs. But for now, I think it would be too confusing and unhelpful. This is definitely a thing we can come back to later though.

Reply:
> we're at that level of efficiency yet with Sorbet

Can you elaborate on that?

Reply:
Run `brew typecheck` without any arguments and you'll see `Errors: 18`. Unless we go forwards and fix them (we're trying, but some constructs for example splats, Sorbet doesn't support very well yet so we'd have to do weird workarounds), or go backwards and mark those files as `typed: false`, then we'll just be confusing people. And look at the length of `sorbet/files.yaml` for what a long way we have to go in the future!

I think we do need to think up a strategy for evolving Sorbet's takeup in Homebrew - I'll chat to Misty and Vidushee about that, and surface it as a thing at the next maintainer call.

Reply:
`--update` seems more appropriate IMO since we're updating tapioca defintions and `files.yaml`.

Reply:
I think it's worth keeping separate for now, I agree. `--update` or `--fix` both sound good to me (with slight preference for `--update`) 👍🏻